lib/chess_web/templates/game/index.html.eex

```diff
@@ -3,10 +3,14 @@
 <table class="table">
   <tbody>
     <%= for game <- @games do %>
-      <tr>
+      <tr class="<%= if your_turn?(@conn, game), do: "your-turn" %>">
```

Comment:
What do you think about extracting another function here displaying the proper class?

```elixir
defp turn_class(@conn, game), do
  if your_turn do
    "your-turn"
  end
end
```
I think it's nice to limit the logic in the template where possible.

Alternatively, in another project, we created a function that follows the same pattern as the elm `classList` function: http://package.elm-lang.org/packages/elm-lang/html/2.0.0/Html-Attributes#classList

```elixir
  def class_list(class_options) do
    class_options
    |> List.wrap()
    |> Enum.filter(fn
      class when is_binary(class) -> true
      {_class, value} -> value
    end)
    |> Enum.map(fn
      class when is_binary(class) -> class
      {class, _} -> class
    end)
    |> Enum.join(" ")
    |> case do
      "" -> nil
      class -> class
    end
  end

```

this gives you an interface like:
```elixir
<tr class="<%= class_list([{"your-turn", your_turn(@conn, game)}]) %>">
```

Just suggestions, no strong opinions! :smile: